File: Problems/seats.py
T = int(input()) 
grp = 0
while(T):
    N = int(input())
    grp = 0
    prev = 0
    while(N):
        cur = N%10
        if((prev==0) and (cur==1)):
            grp = grp+1
        prev = cur
        N = N/10
    print(grp)
    T = T-1

File: Problems/seats.cpp
#include<iostream>
using namespace std;

int main() 
{
    int T;
    cin>>T;
    while(T--)
    {
        string str;
        int prev=0,grp;
        getline(cin,str);
        for(auto i:str)
        {
            if(i=='1' && prev==0)
                grp++;
            prev = i;
        }
        // cout<<grp<<endl;
    }
}